examples/theano/robo_cartpole.py

```diff
@@ -0,0 +1,42 @@
+import gym, roboschool
+import numpy as np
+import random
+
+class NaivePolicy:
```

Comment:
You can obtain random actions directly from the environment obtained from gym by calling:
`env.action_space.sample()`
Therefore, you can remove this class.